src/common.ts

```diff
@@ -0,0 +1,8 @@
+//** TODO: Resolve duplication with Flow */
+export const GLYPH_PROPS_VALID_TYPES: Record<string, Record<string, string>> = {
```

Comment:
What is this needed for? Looks really strange. Better not to touch at all, than having it half done

Reply:
I think let's remove this in the next PR too. I don't like it either.